jurity/recommenders/base.py

```diff
@@ -13,10 +13,14 @@
 
 class _BaseRecommenders(abc.ABC):
     def __init__(self, user_id_column: str = Constants.user_id,
-                 item_id_column: str = Constants.item_id):
+                 item_id_column: str = Constants.item_id,  n_items: Union[int, str] = None,
```

Comment:
If these new args are optional, we can probably make their types optional as well.